data/mods/BombasticPerks/perks.json

```diff
@@ -8,6 +8,8 @@
     "id": "perk_perk_menu",
     "name": { "str": "Open Perk Menu" },
     "points": 0,
+    "purifiable": false,
```

Comment:
I am guessing this was a hiccup? Having two lines of "purifiable"?

Reply:
I was positive I deleted that, but not positive enough I guess

Reply:
Also, surprising that no one ever reported the bug that they purified away their "open perk menu" mutation.